docs/devguide.md

```diff
@@ -31,11 +34,11 @@ layout:
     │   └── hack                # Non-build related scripts
     │   └── jenkins             # Jenkins configuration
     │   └── pkg                 # Contrib golang code
-    ├── charts                  # Helm charts for deployment
-    │   └── catalog             # Helm chart for deploying the catalog
-    │   └── ups-broker          # Helm chart for deploying the user-provided service broker
     ├── docs                    # Documentation
     ├── pkg                     # Contains all non-"main" Go packages
+    ├── plugin                  # Plugins for API server
+    ├── scripts                 # Push images to registry
```

Comment:
@duglin I agree that it's weird we have that script sitting all alone in an ambiguous top-level directory. I think the best place would be to have it in a new `contrib/travis` directory, as there's precedent with the Jenkins scripts. I view `build/` as more of a place for core Service Catalog Dockerfiles and scripts used by `make` during the build. This script is instead used specifically by Travis to trigger a deployment to quay.io.

It seems out of scope for this particular PR, so I'll make a PR for it. :slightly_smiling_face: